include/boost/geometry/arithmetic/cross_product.hpp

```diff
@@ -90,6 +90,12 @@ struct cross_product<3>
 \param p1 first vector
 \param p2 second vector
 \return the cross product vector
+
+
+
+\qbk{[heading Examples]}
+\qbk{[cross_product] [cross_product_output]}
```

Comment:
AFAICT, your ` doc/src/examples/arithmetic/cross_product.cpp` passes homogeneous vector types, that is both arguments and returntype is `P`, it calls this function

```cpp
template <typename P>
inline P cross_product(P const& p1, P const& p2)
```

I'd move these two lines pulling the example in to near that function, near line 119.

Reply:
I've moved the lines.